tinygrad/runtime/ops_torch.py

```diff
@@ -7,8 +7,10 @@
 
 device = torch.device("cuda:0" if torch.cuda.is_available() else ("mps" if getenv("MPS", 0) else "cpu"))
 
+type_map = {dtypes.float16: torch.float16, dtypes.float32: torch.float32}
```

Comment:
I defined the dtypes here instead of in helpers.py, because it requires you to import torch, and it's not a required package. The ImageNet compile test will fail because of this.